indico/modules/events/registration/templates/management/regform.html

```diff
@@ -93,4 +93,9 @@
             </div>
         </div>
     </div>
+    <div class="toolbar right">
+        <a class="i-button icon-chart" href="{{ url_for('.regform_stats', regform) }}">
+            {% trans %}Results{% endtrans %}
```

Comment:
Wouldn't "Stats" be better? "Results" makes sense for a survey but in a registration it sounds strange